docs/configuration.md

```diff
@@ -732,14 +745,14 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 <tr>
     <td><code>spark.executor.heartbeatInterval</code></td>
-    <td>10000</td>
+    <td>10000ms</td>
     <td>Interval (milliseconds) between each executor's heartbeats to the driver.  Heartbeats let
```

Comment:
If the units are now specified in the value, "(millseconds)" can be removed? and is it better to write 10s?